versions/changelogs/0_4_0.md

```diff
@@ -1,8 +1,9 @@
-### Features
-
-* Error message for empty tables
-* Ready for docker setup
-
-### Bugs
-
+### Features
+
+* Error message for empty tables
+* Ready for docker setup
+* Metadata upload and download via API

```

Comment:
```suggestion
* Metadata upload and download via API (#461)
```

Reply:
@wingechr - the version 0.4.0 is already released, you should problably write this in the https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/versions/changelogs/current.md